taxonomies/origins.txt

```diff
@@ -196,16 +196,12 @@ zh: 未指定，不可用，未指定
 # World regions
 
 en: Africa
-af: Afrika
+xx: afrika
```

Comment:
Shouldn’t it be capital-A “Afrika”? (and similar for other entries)

Reply:
This is not case sensitive
This being said, I updated to have upper case everywhere. To be coherant with the rest of the taxonomy.